### PR TITLE
Adding Add button database

### PR DESCRIPTION
**Pull request by ThiemJson**
Pls, code clean

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,5 @@
 Project:  Add foods to Foods-Table. ( render on HTML attr) (owner: ThiemJson)
 
 s1: Create table, button and input-form
-s2: Creat file JS suppport to ADD button
```
Comment (ThiemJson) [inline README.md:4]: Delete make sure you are

File: db.json [hunk 2]
```diff
@@ -1,6 +1,11 @@
 {
-    "posts": [
-      { "id": 1, "title": "json-server", "author": "typicode" }
+    "foods": [
+      { "id": 1, "name": "Rice", "author": "typicode" },
+      { "id": 2, "name": "Noodles", "author": "typicode" },
+      { "id": 3, "name": "Steak", "author": "typicode" },
+      { "id": 4, "name": "Something about rice" , "author": "typicode" },
+      { "id": 5, "name": "Fish ", "author": "typicode" },
+      { "id": 6, "name": "Fruit", "author": "typicode" }
```
Comment (ThiemJson) [inline db.json:8]: change pls author to cost , :) .

File: index.html [hunk 3]
```diff
@@ -5,13 +5,14 @@
...
+    <script src="https://unpkg.com/axios/dist/axios.min.js"></script>
 </head>
 <body>
     <input type="text" id="foods-input">
     <button id="btn-add-foods">Add</button>  
     <table id="foods-table"></table>
 
 
-<script src="./main.js"></script>
+    <script src="./main.js"></script>
```
Comment (ThiemJson) [inline index.html:16]: Dont break line .

File: main.js [hunk 4]
```diff
@@ -1,5 +1,26 @@
...
 var foodTable = document.getElementById("foods-table");
+var url = "http://localhost:9081/foods"
 
+//Bind Event
+btnAdd.addEventListener('click',addItem);
 
+
+//Declare Funciton
+function addItem(){
+
```
Comment (ThiemJson) [inline main.js:12]: what the fuk is that ??? . wtd are you doing ?

**Review by ThiemJson — COMMENTED** [review]
PlSSSSSSSSS

File: README.md (hunk 1 above)
Comment (nguyencaothiem) [reply README.md:4]: I dont think so, it none nessessery

File: db.json (hunk 2 above)
Comment (nguyencaothiem) [reply db.json:8]: i dont think so, wy we so close :)

File: index.html (hunk 3 above)
Comment (nguyencaothiem) [reply index.html:16]: The fuk was that, are u kidding me, i dont think so

File: main.js (hunk 4 above)
Comment (nguyencaothiem) [reply main.js:12]: i supported for u,

**Comment by nguyencaothiem** [comment]
fuk you

File: README.md (hunk 1 above)
Comment (ThiemJson) [reply README.md:4]: PLsss,s wy

**Pull request merged**